### PR TITLE
Fixes #27

### LINKED ISSUE #27: Revert to normal scroll when switching routes.
I have a simple landing page that uses `react-router `, and I'm having issues when navigating away from Fullpage.js.

My render method looks like this, and `react-fullpage` is only active in the Fullpage component:
```javascript
render(
  <Router history={browserHistory}>
    <Route path="/" component={App}>
      <IndexRoute component={Fullpage}/>
      <Route path="/terms" component={Terms}/>
      <Route path="*" component={Fullpage}/>
    </Route>
  </Router>,
  document.getElementById('root')
);
```

When I have `scrollBar: false`, the `Terms` component loads when I navigate to `/terms` but the page scroll is disabled.  When I have `scrollBar: true`, I am able to scroll on the `/terms` route but it initially loads with the same scroll position that the Fullpage route had.

Is there any way to fix this? Either on my end or with a PR to the project? Has anyone else experienced something like this?

### PR DESCRIPTION
**Pull request by 0xpatrickdev**
This pull request fixed the issue I described in #27.

When the component mounts (and `scrollBar` is set to `false`), it sets the overflow property of the body element to `hidden`. I set the overflow back to `initial` when the component unmounts.

### CODE REVIEW

File: dist/SectionsContainer.js [hunk 1]
```diff
@@ -365,7 +371,7 @@ var SectionsContainer = function (_React$Component) {
                     transform: _this5.state.activeSection === index ? 'scale(1.3)' : 'none'
                 };
 
-                return React.createElement('a', { href: '#' + link, key: index, className: (_this5.state.activeSection===index) ? 'Navigation-Anchor '+_this5.props.activeClass : 'Navigation-Anchor',
+                return React.createElement('a', { href: '#' + link, key: index, className: _this5.props.navigationAnchorClass || 'Navigation-Anchor',
```
Comment (0xpatrickdev) [inline dist/SectionsContainer.js:374]: Not sure what is happening here. I only expected changes at lines 59 and 158-162, but this code was also changed during the build process.

**Comment by 0xpatrickdev** [comment]
Please note this only works when all components are wrapped in a `ScrollToTopOnMount` component, as described in [this comment](https://github.com/subtirelumihail/react-fullpage/issues/27#issuecomment-309627780).

The documentation should probably be updated to include a section on how to use this implementation with `react-router`. I can submit another pull request if you think it'd be helpful.

File: dist/SectionsContainer.js (hunk 1 above)
Comment (subtirelumihail) [reply dist/SectionsContainer.js:374]: This is the compiled code of the project

**Comment by subtirelumihail** [comment]
@pcooney10 Can you also add the ScrollToTopOnMount component and update the documentation on this PR so we can push this ?

File: dist/SectionsContainer.js (hunk 1 above)
Comment (0xpatrickdev) [reply dist/SectionsContainer.js:374]: Yes but I didn't change anything with the Navigation Anchors.

Comment (0xpatrickdev) [reply dist/SectionsContainer.js:374]: Looks like this is from c8f673b / #25

**Comment by 0xpatrickdev** [comment]
@subtirelumihail should be good to go now

**Comment by subtirelumihail** [comment]
LGTM 👍 Nice job 😄

**Pull request merged**